internal/pkg/aws/s3/s3.go

```diff
@@ -56,28 +54,6 @@ func New(s *session.Session) *S3 {
 	}
 }
 
-// ZipAndUpload zips all files and uploads the zipped file to an S3 bucket under the specified key.
-// Per s3's recommendation https://docs.aws.amazon.com/AmazonS3/latest/userguide/about-object-ownership.html:
-// The bucket owner, in addition to the object owner, is granted full control.
-func (s *S3) ZipAndUpload(bucket, key string, files ...NamedBinary) (string, error) {
```

Comment:
yay! thank you for deleting dead code <img src="https://emojis.slackmojis.com/emojis/images/1643515051/10804/meow_love.png?1643515051" width="15px" />